scripts/monitoring/cron-send-rpms-versions.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+# Check Docker version
+DOCKER_VERSION=$(docker --version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+HOSTNAME=hostname --fqdn
+
+# Send results to zabbix & hawkular
+ops-metrics-client -s $HOSTNAME -k "docker.version" -o "$DOCKER_VERSION" -t miq_metric="true"
+ops-metrics-client -s $HOSTNAME -k "oc.version" -o "$OC_VERSION" -t miq_metric="true"
```

Comment:
The metrics are sent to zabbix and/or hawkular according to config.
the tag miq_metric="true" is used in manageiq to filter from other metrics in hawkular, for the ones we want to collect and place as node attributes.
This tag of course is not sent to zabbix.

In general I believe this tag should be a default for all metrics, in the metric_sender.yaml, but we can add this later if we decide.